### PR TITLE
Changed the Search on Youtube link to  the new standard naming conven…

### LINKED ISSUE #1552: Change match search query
https://www.facebook.com/groups/moardata/1145277375492556/

### PR DESCRIPTION
**Pull request by technonerdz**
…tion.

Changed the Search on Youtube link to  the new "standard" naming convention.

Closes #1552

### CODE REVIEW

**Comment by fangeugene** [comment]
`match.key_name` includes the event code, which usually isn't included in the naming. I believe the year and event names are more common.

**Comment by technonerdz** [comment]
I tought this was easy but it seems like we don't have a variable with the new "standard" naming convention and I can't make it myself.

File: templates_jinja2/match_details.html [hunk 1]
```diff
@@ -53,7 +53,7 @@
       {% endif %}
       <p>
         <a class="btn btn-success" href="/suggest/match/video?match_key={{match.key_name}}" target="_blank"><span class="glyphicon glyphicon-plus"></span> Add videos</a>
-        <a class="btn btn-primary" href="https://www.youtube.com/results?search_query={{match.verbose_name|urlencode}}+{{event.year}}+{{event.name|urlencode}}" target="_blank"><span class="glyphicon glyphicon-search"></span> Search YouTube</a>
+        <a class="btn btn-primary" href="https://www.youtube.com/results?search_query={{match.key_name}}" target="_blank"><span class="glyphicon glyphicon-search"></span> Search YouTube</a>
```
Comment (phil-lopreiato) [inline templates_jinja2/match_details.html]: The current format is `<match type>(<set>)M<match num> <event name>`, see [this example](https://www.youtube.com/user/NewEnglandFIRST/playlists)

You can build that with `{{match.short_name|urlencode}}+{{event.name|urlencode}}`

See the definition of [match.short_name](https://github.com/the-blue-alliance/the-blue-alliance/blob/master/models/match.py#L176)

**Comment by phil-lopreiato** [comment]
@technonerdz there's no reason to close this pull, you were on the right track

**Comment by technonerdz** [comment]
Sorry! I taught that this would be more difficult and I didn't saw the variable and I was ready to let someone else do it. Is it ok now?

**Comment by phil-lopreiato** [comment]
👍 LGTM

**Pull request merged**